packages/website/ts/components/top_bar/provider_display.tsx

```diff
@@ -66,21 +61,21 @@ export class ProviderDisplay extends React.Component<ProviderDisplayProps, Provi
             this.props.providerType === ProviderType.Injected ? injectedProviderName : 'Ledger Nano S';
         const isProviderMetamask = providerTitle === constants.PROVIDER_NAME_METAMASK;
         const hoverActiveNode = (
-            <div className="flex right lg-pr0 md-pr2 sm-pr2 p1" style={styles.root}>
+            <div className="flex items-center p1" style={styles.root}>
                 <div>
                     {this._isBlockchainReady() ? (
                         <Identicon address={this.props.userAddress} diameter={ROOT_HEIGHT} />
                     ) : (
                         <CircularProgress size={ROOT_HEIGHT} thickness={2} />
                     )}
                 </div>
-                <div style={{ marginLeft: 12, paddingTop: 3 }}>
-                    <div style={{ fontSize: 16, color: colors.darkGrey }}>{displayMessage}</div>
-                </div>
+                <Container marginLeft="12px" marginRight="12px">
```

Comment:
fwiw I'm a bit conflicted about `Container` in the context of us having bass.css with all the padding / margin utility classes. I think the argument for `Container` is that it makes components more portable, and removes our dependency (although we still use the "flex" class heavily) on a global stylesheet. Also having our styles strictly typed is nice. 
OTH I hate having two ways of doing the same thing, and basscss is the prior art here and works quite well.

Nothing actionable here: just wondering what your opinion is...
We could make `Container` support an API like this https://github.com/jxnblk/bass-component but depend on the global stylesheet (ie. props become classNames). Then at least one day we are better set-up for a migration if we were so inclined. I think it would be pretty easy to make Container extend this component (both are made with styled-components)